cmd/operator-sdk/run/cmd.go

```diff
@@ -60,13 +62,10 @@ func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run",
 		Short: "Run an Operator in a variety of environments",
-		Long: `This command will run or deploy your Operator in two different modes: locally
-and using OLM. These modes are controlled by setting --local and --olm run mode
-flags. Each run mode has a separate set of flags that configure 'run' for that
-mode. Run 'operator-sdk run --help' for more information on these flags.
-
-Read more about the --olm run mode and configuration options here:
-https://sdk.operatorframework.io/docs/olm-integration/cli-overview
```

Comment:
Should we really remove the link for the doc which give further information? How users will know what means, I mean when an why the `packagemanifests` subcommand would be used? Or what means `using OLM`?

Reply:
This link is in `run packagemanifests --help` now.